docs/source/quickstart.rst

```diff
@@ -84,7 +80,7 @@ To do this, you can either
 
 1. Install Chaco and its :ref:`dependencies` from `PyPI
    <http://pypi.python.org/pypi>`_ using `easy_install
-   <http://packages.python.org/distribute/easy_install.html>`_ (part of
```

Comment:
ETS _is_ Enthought Tool Suite.  The training courses you are probably confusing with are often referred to as EToD. So to clarify:

* ETS: "Enthought Tool Suite," our collection of open source packages.
* EPD: "Enthought Python Distribution," our old Python distribution, now replaced by "Canopy"
* EToD: "Enthought Training on Demand," our online training courses.

..and so you should leave "ETS" as is, but replace "EPD" with "Canopy".